templates/monster_armor_how_to.html

```diff
@@ -55,23 +59,41 @@ <h2 id="ap_level">AP Level</h2>
                 </div>
 				<div class="col-lg-8 col-md-6">
                     <h2 id="armor_points"> Armor Points </h2>
-                    <p> Text </p>
+                    <p> Armor Points are, effectively, your armor's ability to slow down projectiles as they
+                        pass through it. Whenever armor is successfully pierced, all damage is first inflicted
+                        on APs. Once the APs are used up, all remaining damage goes to health. APs refresh at the
```

Comment:
We might want to consider renaming Armor Points or Armor Piercing/Penetration so that they don't have the same abbreviation.

Reply:
That's a whole 'nother bag of worms to discuss in the Compound_X repo.

Reply:
Issue submitted